.github/workflows/lint.yml

```diff
@@ -204,8 +206,8 @@ jobs:
           path: flake8-output/
       - name: Fail if there were any warnings
         run: |
-          cat flake8-output.txt
```

Comment:
Does this need to be outputting the flake8 results here? They should be done in `Run flake8` above since it's `tee`d